svc/FetchService.js

```diff
@@ -45,11 +45,12 @@ export class FetchService {
      * @param {Object} [opts.qsOpts] - Object of options to pass to the param converter library, qs.
      *      The default qsOpts are: {arrayFormat: 'repeat', allowDots: true}.
      *      @see {@link https://www.npmjs.com/package/qs}
+     * @param {string} [opts.requestKey] - Key to the abort object to call when cancelling an in flight request.
```

Comment:
This doc and name arr now a bit confusing.  It seems to me that you have implemented something that ought to be named 'autoCancelKey' and be described as.
     "If set, any pending requests associated with the same autoCancelKey will
     be immediately cancelled when a new request is made".